src/sentry/api/serializers/models/project.py

```diff
@@ -160,6 +162,14 @@ def serialize(self, obj, attrs, user):
 
         status_label = STATUS_LABELS.get(obj.status, 'unknown')
 
+        if attrs.get('avatar'):
+            avatar = {
+                'avatarType': attrs['avatar'].get_avatar_type_display(),
+                'avatarUuid': attrs['avatar'].ident if attrs['avatar'].file else None
```

Comment:
I think you just want to check `.file_id` instead, which will avoid traversing the relationship. Unclear if you use `.file` anywhere else, but if you don't, then you could ignore my comment about `select_related`.